### DISCUSSION TITLE
send email failly

### DISCUSSION THREAD

**Discussion by tuohaibei** [abpframework/abp#19229] [General] [answered]
Is anyone have the same problem?
![image](https://github.com/abpframework/abp/assets/5082315/fa1f9d89-d42f-44fb-92a5-c73e53916c8a)

**Comment by maliming** [comment]
You can test email sender via https://github.com/abpframework/abp-samples/tree/master/EmailSendDemo

**Reply by tuohaibei** [reply]
Hi, i know why it have the problem,i have readed the config ,it seems it is not right, but database is right,but i don't know why it could not  read database
![image](https://github.com/abpframework/abp/assets/5082315/1c364478-38b5-4bc8-80e6-8b5235d2fc02)

![image](https://github.com/abpframework/abp/assets/5082315/54cd5e29-c3c5-4556-b217-a1550deac632)

**Reply by tuohaibei** [reply, marked as answer]
I have fixed the problem,beacuse it read configuration from redis,i clear redis and it is ok.
